src/net.cpp

```diff
@@ -2532,7 +2532,7 @@ bool CConnman::Start(CScheduler& scheduler, const Options& connOptions)
     if (fListen && !InitBinds(connOptions)) {
         if (m_client_interface) {
             m_client_interface->ThreadSafeMessageBox(
-                _("Failed to listen on any port. Use -listen=0 if you want this."),
+                _("Failed to listen on any port.\nUse -listen=0 if you want this."),
```

Comment:
I think we're generally trying to avoid having newlines in translation messages? @hebasto

Reply:
Yes. As I mentioned in https://github.com/bitcoin-core/gui/pull/476#issuecomment-974891494, we cannot be sure that the chosen location of the hardcoded `\n` is optimal for translated strings.

If the problem is the breaking `-listen=0`, why do not use a nonbreaking hyphen:
```suggestion
                _("Failed to listen on any port. Use \u2011listen=0 if you want this."),
```
?

Reply:
Breaking text at the end of the sentence would be cutest.
(at least at that macOS build).
Breaking text before `-listen=0` is good enough as a first step.